### DISCUSSION TITLE
Practices for Running Hasura at different URLs or Paths

### DISCUSSION THREAD

**Discussion by jeremiahrhall** [hasura/graphql-engine#6257] [Question]
I would like to be able to set a path prefix for Hasura such as `/myservice`. For example, ideally I could run hasura at URLs like `mydomain.com/myservice/console` or `mydomain.com/myservice/v1/graphql`.

I noticed that Hasura Cloud provisions instances at subdomains such as `https://something.hasura.app/v1/graphql` so y'all have adopted subdomains yourselves.

I can do this with Nginx rewrites, but was wondering if you all had any suggested ways of accomplishing this.

**Comment by lwsbox** [comment]
Now is 2023.
